internal/service/athena/workgroup_test.go

```diff
@@ -686,6 +686,38 @@ func TestAccAthenaWorkGroup_tags(t *testing.T) {
 	})
 }
 
+func TestAccAthenaWorkGroup_ManagedQueryResultsConfiguration(t *testing.T) {
```

Comment:
Can we add additional steps to this test case to exercise disabling, then re-enabling the feature to ensure update works as expected?